source/intro.tex

```diff
@@ -424,8 +424,8 @@
 
 \pnum
 Throughout this International Standard, each example is introduced by
-``\enterexample'' and terminated by ``\exitexample''. Each note is
-introduced by ``\enternote'' and terminated by ``\exitnote''. Examples
+``\begin{example}'' and terminated by ``\end{example}''. Each note is
```

Comment:
@godbyk: Thank you, but you can leave it for now; @jwakely has an up-to-date copy of the branch that incorporates some additional adjustments needed to go with this change. Hopefully we'll get it in soon. Thank you very much for preparing this large improvement!